app/src/androidTest/java/org/mozilla/focus/telemetry/TelemetryWrapperTest.java

```diff
@@ -0,0 +1,579 @@
+package org.mozilla.focus.telemetry;
+
+import android.content.Context;
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.preference.PreferenceManager;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mozilla.focus.R;
+import org.mozilla.focus.activity.MainActivity;
+
+import static org.mozilla.focus.telemetry.TelemetryWrapper.Value.AUDIO;
+import static org.mozilla.focus.telemetry.TelemetryWrapper.Value.EME;
+import static org.mozilla.focus.telemetry.TelemetryWrapper.Value.MIDI;
+import static org.mozilla.focus.telemetry.TelemetryWrapper.Value.VIDEO;
+
+@RunWith(AndroidJUnit4.class)
+public class TelemetryWrapperTest {
+
+    @Rule
+    public ActivityTestRule<MainActivity> activityTestRule = new ActivityTestRule<>(MainActivity.class, true, false);
+
+    @Before
+    public void setUp() {
+        final Context context = InstrumentationRegistry.getInstrumentation().getTargetContext();
+        final String prefName = context.getString(R.string.pref_key_telemetry);
+        final SharedPreferences preferences = PreferenceManager.getDefaultSharedPreferences(context);
+        preferences.edit().putBoolean(prefName, false).apply();
+
+    }
+
+
+    @Test
```

Comment:
What are the purposes of these three methods?

Reply:
I'll remove unused method later.